docs/topics/http/urls.txt

```diff
@@ -51,19 +50,19 @@ algorithm the system follows to determine which Python code to execute:
 3. Django runs through each URL pattern, in order, and stops at the first
    one that matches the requested URL.
 
-4. Once one of the regexes matches, Django imports and calls the given view,
-   which is a simple Python function (or a :doc:`class-based view
+4. Once one of the URL patterns matches, Django imports and calls the given
+   view, which is a simple Python function (or a :doc:`class-based view
    </topics/class-based-views/index>`). The view gets passed the following
    arguments:
 
    * An instance of :class:`~django.http.HttpRequest`.
-   * If the matched regular expression returned no named groups, then the
+   * If the matched URL pattern returned no named groups, then the
```

Comment:
This is specific to `url`/`re_path`, because `path` does not allow for unnamed groups.